ansible/configs/smart-management/post_software.yml

```diff
@@ -16,20 +16,17 @@
 - name: Import Tower License from Bastion 
   hosts: bastions
   become: false
+  collections:
+    - awx.awx
   tasks:
-    - name: Import Tower License With API
-      uri:
-        url: https://{{ tower_hostname }}/api/v2/config/
-        user: admin
-        password: "{{ tower_admin_password }}"
-        method: POST
-        headers:
-          Content-Type: "application/json"
-          Accept: "application/json"
-        body: "{{ tower_license }}"
-        body_format: json
-        force_basic_auth: yes
-        validate_certs: false
+    - name: Set the license using a file
+      tower_license:
```

Comment:
Use `awx.awx.tower_license`, but this cannot be directly in the playbook, needs to be behind a `include_tasks` or `include_role`